src/block/timeline/edit.js

```diff
@@ -86,14 +91,31 @@ const Edit = props => {
 		className,
 		clientId,
 		isSelected,
+		setAttributes,
 	} = props
 
 	useGeneratedCss( props.attributes )
 
 	const rowClass = getRowClasses( props.attributes )
 	const separatorClass = getSeparatorClasses( props.attributes )
 	const blockAlignmentClass = getAlignmentClasses( props.attributes )
+	const typographyClass = getTypographyClasses( props.attributes )
 	const { hasInnerBlocks } = useBlockContext()
```

Comment:
You can now do this to get the `nextBlock` and `previousBlock` - you can remove the manual getting of next and previous elements in your `getAdjacentBlocks`
```suggestion
	const {
		hasInnerBlocks, nextBlock, previousBlock,
	} = useBlockContext()
```